build.gradle.kts

```diff
@@ -70,6 +80,39 @@ allprojects {
         }
 
         tasks {
+            withType<JavaCompile> {
+                options.errorprone {
+                    error(
+                            "AssertionFailureIgnored",
+                            "BadInstanceof",
+                            "BoxedPrimitiveConstructor",
+                            "CatchFail",
+                            "DefaultCharset",
+                            "InheritDoc",
+                            "MathAbsoluteRandom",
+                            "MissingOverride",
+                            "UnnecessaryParentheses"
+                    )
+
+                    // TODO(anuraaga): These will improve the Javadoc but punt for now since it's a lot of work and not
+                    // as important as code checks.
+                    disable(
+                            "EmptyBlockTag",
+                            "MissingSummary"
```

Comment:
Once enabled, will this conflict with or provide duplicate messages with the JavaDoc plugin?